jetty-util/src/main/java/org/eclipse/jetty/util/thread/ShutdownThread.java

```diff
@@ -36,10 +37,11 @@
 public class ShutdownThread extends Thread
 {
     private static final Logger LOG = LoggerFactory.getLogger(ShutdownThread.class);
-    private static final ShutdownThread _thread = new ShutdownThread();
+    private static final ShutdownThread INSTANCE = new ShutdownThread();
```

Comment:
Again, this might strictly be correct, but should have only been changed after the synchronization is converted to locks.